examples/device/simple_spmv.cpp

```diff
@@ -0,0 +1,66 @@
+#include <iostream>
+#include <spblas/spblas.hpp>
+
+#include <thrust/device_vector.h>
+
+#include <fmt/core.h>
+#include <fmt/ranges.h>
+
+int main(int argc, char** argv) {
+  using namespace spblas;
```

Comment:
I am not a big fan of 'using's. I prefer spblas::multiply

Reply:
after checking, realiasing the multiply is a bit annoying due to the template. I use the spblas::multply directly.